feat: generate mnemonic utility

Markeljan opened the pull request:
A generate mnemonic bun script for creating a random mnemonic using the existing @stacks/wallet-sdk package.  (for devs not agent)

whoabuddy commented on README.md:
```diff
@@ -12,4 +12,6 @@ curl -fsSL https://bun.sh/install | bash
 
 3. Install dependencies: `bun install`
-4. Run scripts: `bun run <script-name>`
+4. Copy `.env.example` to `.env` and fill in the required environment variables : `cp .env.example .env`
+5. (Optional) Generate a new mnemonic: `bun run generate-mnemonic`
```
Should these be reversed? Setup wallet first, then setup `.env`?

```suggestion
4. (Optional) Generate a new mnemonic: `bun run generate-mnemonic`
5. Copy `.env.example` to `.env` and fill in the required environment variables : `cp .env.example .env`
```

whoabuddy commented on package.json:
```diff
@@ -1 +1,8 @@
\ No newline at end of file
+{
+  "scripts": {
+    "generate-mnemonic": "node -e 'import(\"@stacks/wallet-sdk\").then(({generateSecretKey}) => console.log(generateSecretKey()))'"
```
This is an interesting approach, right now each script lives as an individual TS file so would've expected it to match the format.

Bun can execute the TS file just like `ts-node` and install dependencies based on the package.json.

Curious why you used the package.json scripts for the functionality instead?

Markeljan replied:
Since the current scripts are all intended for the agent to use I didn't want to mix in one that is made for the developer.  However this could in theory be used by the agent at some point so maybe ti can be?

whoabuddy commented on tsconfig.json:
Weird I thought this was in there, good catch!

whoabuddy reviewed (changes requested):
See comments, this is looking good just a few small things to address.

Markeljan commented:
@whoabuddy I added generateMnemonic to `utilities.ts` and switched to a bunx script that uses run-func npm package.

Seems like a good way to have a util for both developer and agents to use instead of having to create a standalone .ts file just for one task.

LMK thoughts or feel free to implement however you see fit
